### DISCUSSION TITLE
[@types/glob] Error: Cannot find module 'glob'

### DISCUSSION THREAD

**Discussion by Wilson159** [DefinitelyTyped/DefinitelyTyped#61973] [Issues with a @types package] [answered]
I have [@types/glob] installed with npm yet I'm still getting the following error: https://pastebin.com/VDNyTz9F...

[a comment by an automated account is not shown]

**Comment by BendingBender** [comment]
Did you install the `glob` module additionally to the `@types/glob`?

You should run something like this:
```sh
npm install glob
npm install --save-dev @types/glob
```

**Comment by peterblazejewicz** [comment]
@Wilson159 not sure the syntax and setup, just ensure you've 'esModuleInterop' set to `true` in your tsconfig

**Comment by Wilson159** [comment, marked as answer]
Thanks, guys I got it to work now, I removed all of my dependencies and reinstalled everything...
